### PR TITLE
First CI job for kubernetes/cloud-provider-openstack

### PR DESCRIPTION
**Pull request by dims**

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -392,6 +392,26 @@ presubmits:
...
+    context: pull-cloud-provider-openstack-test
+    rerun_command: "/test pull-cloud-provider-openstack-test"
+    trigger: "(?m)^/test( all| pull-cloud-provider-openstack-test),?(\\s+|$)"
+    branches:
+    - master
+    labels:
+      preset-service-account: true
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
```
Comment (BenTheElder) [inline prow/config.yaml]: it *might* be better to use kubekins-e2e to match a kubernetes release branch's go version.

Comment (krzyzacy) [reply prow/config.yaml]: warning: this image is pinned to gcloud 163.0.0 :-)

Comment (dims) [reply prow/config.yaml]: ack. switching to `gcr.io/k8s-testimages/kubekins-e2e:v20180316-92e6ccade-master`

**Comment by dims** [comment]
i fixed the image. please reapply `/lgtm` and i'll kick the tires

**Comment by krzyzacy** [comment]
assume you'll need to enable trigger from prow/plugins.yaml

**Pull request merged**